src/types/WriteStream.ts

```diff
@@ -0,0 +1,4 @@
+export interface WriteStream {
+  write: (content: string, callback: (err?: Error) => any) => void
```

Comment:
Replaced the import of `fs.WriteStream` with our own, locally defined interface.